src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -18,27 +18,31 @@ export default function ColorPickerControl({
     ...rest
 }: Props): JSX.Element | null {
     const paletteRef = React.useRef<HTMLDivElement>(null);
+    const toggleRef = React.useRef<HTMLButtonElement>(null);
     const [isColorPickerToggled, setIsColorPickerToggled] = React.useState(false);
-    const [isPaletteActive, handlers] = useAttention();
+    const [handlers] = useAttention();
```

Comment:
Do we still need `useAttention` if we're not using the state it returns?

Reply:
Good point, will remove